gapic/src/main/com/google/gapid/perfetto/views/TraceConfigDialog.java

```diff
@@ -295,6 +295,12 @@ public static String getConfigSummary(Settings settings, Device.PerfettoCapabili
             .getConfigBuilder()
                 .setName("android.surfaceflinger.frame");
       }
+      // Skip UI checkbox for GpuMemTotal producer to align with its ftrace events
```

Comment:
I don't understand this comment. Are you trying to explain why we don't allow the user to disable this? If so, can you phrase it like this: "Always enable GPU memory, if available, because [insert reason here]"

Reply:
Done.